docs/extensions/extension_example.md

```diff
@@ -106,7 +102,8 @@ this function is asynchronous, we need to `await` it.
 
 The lockfile object will look something like this:
 
-```text
+```js
```

Comment:
This applies to all the codeblocks that were changed in this file:

I'm not sure calling these `js` is the best idea. I've intentionally marked them as `text` because they represent the text output of `console.log` for these structs. They might technically be in a format that is a valid JavaScript object, but it's not really JavaScript code.

Reply:
Okay...they will be changed back.